### PR TITLE
Updates in the  migration guide - Closes #1141

### LINKED ISSUE #1141: Updates in the migration guide
- add the `--interpreter` param to the post-install pm2 section in Core setup
- add `pm2 save` command to the docs everywhere where it is about management with pm2
like here: https://lisk.com/documentation/lisk-service/management/source.html
and here: https://lisk.com/documentation/lisk-core/v3/management/pm2.html
- Add steps how to vote with multi-signature accounts
```
lisk-core transaction:create 5 1 10000000
lisk-core transaction:sign hex-output --mandatory-keys="second sig pk1" mandatory-keys="second sig pk2" --sender-public-key="original-pk"
lisk-core transaction:sign hex-output-from-above --mandatory-keys="second sig pk1" mandatory-keys="second sig pk2" --sender-public-key="original-pk"
```
- step 2.6: `pm2.conf.json` replace ~ with /home/lisk/

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1141

### CODE REVIEW

File: modules/ROOT/pages/management/migration.adoc [hunk 1]
```diff
@@ -547,6 +547,44 @@ $ lisk-core transaction:send 0805100118012080c2d72f2a2024350a05e078b181fa8f3c273
 Transaction with id: '6a6121adf6a73a857bef92eaec9c29545f53c9196a16faa1eafdf58012f5a2e5' received by node.
 ----
 
+==== Voting with a multiginature account
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/migration.adoc]:
```suggestion
==== Voting with a multisignature account
```

File: modules/ROOT/pages/management/migration.adoc [hunk 2]
```diff
@@ -47,7 +47,7 @@ OS::
 * Ubuntu 20.04
 * Ubuntu 18.04
 
-If you have Node.js installed ensure to use 12.22.1.
+If you have Node.js installed ensure to use 12.22.2.
```
Comment (ghost) [inline modules/ROOT/pages/management/migration.adoc]: 12.22.3 is out already. would it make sense to replace this exact version in the documentation by `12.22.x` or even `12.x`?

File: modules/ROOT/pages/management/migration.adoc [hunk 3]
```diff
@@ -547,6 +547,44 @@ $ lisk-core transaction:send 0805100118012080c2d72f2a2024350a05e078b181fa8f3c273
 Transaction with id: '6a6121adf6a73a857bef92eaec9c29545f53c9196a16faa1eafdf58012f5a2e5' received by node.
 ----
 
+==== Voting with a multiginature account
+
+All accounts which had a second passphrase registered in Lisk Core v2 are converted to multisignature accounts which need 2 mandatory signatures.
+The voting process therefore works a little different:
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/migration.adoc]:
```suggestion
The voting process therefore works a little different for accounts with second passphrase:
```

File: modules/ROOT/pages/management/migration.adoc [hunk 4]
```diff
@@ -547,6 +547,44 @@ $ lisk-core transaction:send 0805100118012080c2d72f2a2024350a05e078b181fa8f3c273
...
+Transaction with id: 'cf1e7af1b385a630c893ac45c9a4cdf5536769ba6e678a9779ff9af135ee5ec9' received by node.
+----
+
+<1> Use the *passphrase* of the *first* account.
+<2> Use the *public key* of the *first* account.
+<3> Use the *public key* of the *second* account.
+<4> Use the *public key* of the *first* account.
+<5> Use the *passphrase* of the *first* account.
+<6> Use the *passphrase* of the *second* account.
+
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/migration.adoc:590]: also important to specify to copy the output of each sign to next sign function? ppl who know wont make that mistake but its better we specify explicitly?

Comment (Tschakki) [reply modules/ROOT/pages/management/migration.adoc:590]: Good point. I added some new comments in the above code snippet

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**